portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -38,11 +41,13 @@
 public class DataServlet extends HttpServlet {
   
   private class Comment {
```

Comment:
Missed this in a previous review.

Unless you're in a pretty weird situation, it's best to make [nested classes](https://docs.oracle.com/javase/tutorial/java/javaOO/nested.html) `static`, like so:

```java
private static class Comment {
```

The static keyword in this context basically means that this Comment class doesn't need any special access to the DataServlet class (i.e., its instance members). This allows for a clean encapsulation between the two classes.